lua/ibl/config.types.lua

```diff
@@ -42,6 +42,8 @@
 ---@field smart_indent_cap boolean?
 --- Virtual text priority for the indentation guide
 ---@field priority number?
+--- Repeat indentation guide virtual text on wrapped lines if 'breakindent' is set
+---@field repeat_linebreak boolean
```

Comment:
This has to be optional, that's also the reason the LSP tests fail.
```suggestion
---@field repeat_linebreak boolean?
```